src/__tests__/DataShape.test.js

```diff
@@ -0,0 +1,22 @@
+import { data } from '../data/data';
```

Comment:
The { data } object is deprecated which will obviate the viability of these tests. Easy solution is to copy the data object you were importing in for these tests and use it in the test file instead of importing it in from the data.js file.